checkmate/app.py

```diff
@@ -4,43 +4,45 @@
 import pyramid.config
 import pyramid_tm
 
-REQUIRED_PARAMS = ["checkmate_blocklist_path"]
-OPTIONAL_PARAMS = ["database_url"]
 
-
-def load_settings(settings):
+def load_settings(settings, celery_worker=False):
     """Load application settings from a dict or environment variables.
 
     Checks that the required parameters are either filled out in the provided
     dict, or that the required values can be loaded from the environment.
 
     :param settings: Settings dict
+    :param celery_worker: Only do the configuration strictly required for
+        celery workers
     :raise ValueError: If a required parameter is not filled
     :return: A dict of settings
     """
-    for param in OPTIONAL_PARAMS:  # pragma: no cover
-        value = settings.get(param, os.environ.get(param.upper()))
-        if value:
-            settings[param] = value
 
-    for param in REQUIRED_PARAMS:
+    required_params = ["database_url", "checkmate_blocklist_path"]
+    if celery_worker:
+        required_params.append("checkmate_blocklist_url")
+
+    for param in required_params:
         value = settings[param] = settings.get(param, os.environ.get(param.upper()))
 
         if value is None:
             raise ValueError(f"Param {param} must be provided.")
 
     # Configure sentry
     settings["h_pyramid_sentry.filters"] = []
+    if celery_worker:
+        settings["h_pyramid_sentry.celery_support"] = True
 
     return settings
 
 
-def create_app(_=None, **settings):  # pragma: no cover
-    """Configure and return the WSGI app."""
-
-    config = pyramid.config.Configurator(settings=load_settings(settings))
+def configure(config, celery_worker=False):  # pragma: no cover
```

Comment:
Split this out so it could be used in `tests/unit/checkmate/async/tasks_test.py`